packages/fuel-gauge/package.json

```diff
@@ -16,8 +16,9 @@
   },
   "devDependencies": {
     "@fuel-ts/account": "workspace:*",
+    "@fuel-ts/errors": "workspace:*",
     "@fuel-ts/forc": "workspace:*",
     "@fuel-ts/utils": "workspace:*",
-    "@fuel-ts/errors": "workspace:*"
+    "delay": "^6.0.0"
```

Comment:
We could probably move:
 - [packages/account/src/providers/utils/sleep.ts](https://github.com/FuelLabs/fuels-ts/blob/9c3c094d810b8400e8c0b9ac2830f75ca2ca70b5/packages/account/src/providers/utils/sleep.ts)

To:
 - [packages/utils/src/utils](https://github.com/FuelLabs/fuels-ts/tree/9c3c094d810b8400e8c0b9ac2830f75ca2ca70b5/packages/utils/src/utils)

Then, import and use it from there instead `delay`.

```suggestion
```

Reply:
Example:

https://github.com/FuelLabs/fuels-ts/blob/9c3c094d810b8400e8c0b9ac2830f75ca2ca70b5/packages/account/src/providers/utils/auto-retry-fetch.ts#L82